src/OpenApi/src/OpenApiGenerator.cs

```diff
@@ -31,6 +31,76 @@ internal sealed class OpenApiGenerator
 
     internal static readonly ParameterBindingMethodCache ParameterBindingMethodCache = new();
 
+    private static readonly Dictionary<int, string> Phrases = new()
```

Comment:
This is built in https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.webutilities.reasonphrases.getreasonphrase?view=aspnetcore-7.0#microsoft-aspnetcore-webutilities-reasonphrases-getreasonphrase(system-int32)